From 3cb653792528d74d98b904414c02d4464b6cd4ed Mon Sep 17 00:00:00 2001
From: Cristian Ciocaltea <cristian.ciocaltea@collabora.com>
Date: Thu, 11 Jul 2024 14:48:43 +0300
Subject: [PATCH 173/317] FROMEXT: [WIP] arm64: defconfig: Enable Rockchip
 extensions for Synopsys DW HDMI QP

Enable Rockchip specific extensions for the Synopsys DesignWare HDMI QP
driver.  This is needed to provide HDMI output support for boards based
on the RK3588 SoC.

Signed-off-by: Cristian Ciocaltea <cristian.ciocaltea@collabora.com>

Link: https://gitlab.collabora.com/hardware-enablement/rockchip-3588/linux/-/commit/13ce112bae559e6189673ac2f5806dc346e2d116
Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 arch/arm64/configs/defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index c62831e61586..286a38649601 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -872,6 +872,7 @@ CONFIG_ROCKCHIP_VOP2=y
 CONFIG_ROCKCHIP_ANALOGIX_DP=y
 CONFIG_ROCKCHIP_CDN_DP=y
 CONFIG_ROCKCHIP_DW_HDMI=y
+CONFIG_ROCKCHIP_DW_HDMI_QP=y
 CONFIG_ROCKCHIP_DW_MIPI_DSI=y
 CONFIG_ROCKCHIP_INNO_HDMI=y
 CONFIG_ROCKCHIP_LVDS=y
-- 
2.48.1

